diff --git src/index.php src/index.php
index 0247e49..119038b 100644
--- src/index.php
+++ src/index.php
@@ -180,6 +180,12 @@ $app->group('/job', function() use ($app) {
      * Runs a cron job in background.
      */
     $app->get('/run/:hash', $setupJsonResponse, function($hash) use ($app) {
+        $app->render(403, array(
+            'error' => true,
+            'msg' => 'Running cron jobs is disabled in the demo application'
+        ));
+        $app->stop();
+        
         $crontab = new Crontab();
         $job = $crontab->findByHash($hash);
         
